app/src/pages/inside/stepPage/modals/makeDecisionModal/tabs/selectDefectManually/selectDefectManually.jsx

```diff
@@ -88,6 +93,16 @@ export const SelectDefectManually = ({
   };
   const handleDefectCommentChange = (value) => {
     handleManualChange({ comment: value.trim() });
+    if (isBulkOperation) {
+      if (!source.issue.comment && value.trim() !== '') {
+        setModalState({ commentOption: ADD_FOR_ALL });
+      } else if (
+        value.trim() === '' &&
```

Comment:
May be calculate this condition once?
f.e. `const isValueEmpty = value.trim() === '';`.